impeller/tools/impeller.gni

```diff
@@ -700,12 +710,22 @@ template("impeller_shaders") {
                "use_half_textures",
              ])
 
+  enable_metal = impeller_enable_metal
+  if (defined(invoker.enable_metal)) {
+    enable_metal = invoker.enable_metal
+  }
+
   enable_opengles = impeller_enable_opengles
   if (defined(invoker.enable_opengles)) {
     enable_opengles = invoker.enable_opengles
   }
 
-  if (impeller_enable_metal) {
+  enable_vulkan = impeller_enable_vulkan
+  if (defined(invoker.enable_vulkan)) {
+    enable_vulkan = invoker.enable_vulkan
+  }
+
```

Comment:
Consider asserting (enable_metal || enable_opengles || enable_vulkan)

Reply:
Done.

Reply:
Not clear to me why the playground is getting included in the fuchsia builds.

Reply:
Not sure, but wondering if the deps here also need an `impeller_supports_rendering` guard:
https://github.com/flutter/engine/blob/main/BUILD.gn#L184

Reply:
I'm not sure I really understand what's goign on here, but maybe just `(!impeller_supports_rendering || enable_metal || enable_opengles || enable_vulkan)`?